packages/webpack-rtl-plugin/src/index.js

```diff
@@ -36,8 +36,7 @@ class WebpackRTLPlugin {
 								}
 
 								// Compute the filename
-								const baseName = path.basename( asset, '.css' );
-								const filename = asset.replace( baseName, `${ baseName }.rtl` );
+								const filename = asset.replace( cssRe, '.rtl$&' );
```

Comment:
What is this `$&` bit?

Reply:
It inserts the matched substring into the replacement. `cssRe` is going to match either `.css` (at the end of the string) or `.css?` (somewhere in the middle), so the replacement winds up being `.rtl.css` or `.rtl.css?` accordingly.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_string_as_a_parameter

Reply:
TIL, thanks!